cmd/instance_pool_list.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type instancePoolItem struct {
+	ID    *egoscale.UUID             `json:"id"`
+	Name  string                     `json:"name"`
+	Zone  string                     `json:"zone"`
+	Size  int                        `json:"size"`
+	State egoscale.InstancePoolState `json:"state"`
+}
+
+type instancePoolListItemOutput []instancePoolItem
+
+func (o *instancePoolListItemOutput) toJSON()  { outputJSON(o) }
+func (o *instancePoolListItemOutput) toText()  { outputText(o) }
+func (o *instancePoolListItemOutput) toTable() { outputTable(o) }
+
+var instancePoolListCmd = &cobra.Command{
+	Use:     "list [zone]",
+	Short:   "List instance pool",
```

Comment:
The command is missing the `Long` field with custom output annotations, see how other `list` commands are implemented for reference.